sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -475,7 +495,7 @@ private static boolean initConfigurations(final @NotNull SentryOptions options)
     }
 
     if (options.isEnableBackpressureHandling()) {
-      options.setBackpressureMonitor(new BackpressureMonitor(options, HubAdapter.getInstance()));
+      options.setBackpressureMonitor(new BackpressureMonitor(options, ScopesAdapter.getInstance()));
```

Comment:
Could use the `HubScopesWrapper` here to make it compile:
`options.setBackpressureMonitor(new BackpressureMonitor(options, new HubScopesWrapper(ScopesAdapter.getInstance())));`

Reply:
PRs 2-13 will replace the `IHub` interface with `IScopes` so this should work just fine after. I had to split into multiple PRs as this was way too big for review with 200+ files changed.